### DISCUSSION TITLE
Blank screen except for a cursor.

### DISCUSSION THREAD

**Discussion by Si-Ki** [hyprwm/Hyprland#896] [Q&A]
Hyprland version: hyprland-bin form aur `0.16.0 beta-1`
I wasn't sure if this classifies as a bug, so posting this here instead of creating an issue,

I install hyprland, 
I make this [file](https://0x0.st/oxL_.sh), and run it 
hyprland starts, I see cursor, nothing else, 
I can launch alacritty, but can't see it (sure that it launched cuz I wrote "killall Hyprland" and it worked, the screen was black all the time, the cursor changed to a caret)

Log [here](https://0x0.st/oxLQ.log),

sorry for any stupid mistakes I might have made, and thank for helping @vaxerski

**Comment by vaxerski** [comment]
1) Zen kernel, have you tried regular?
2) Why `export LIBSEAT_BACKEND=logind`?
3) launch with `HYPRLAND_LOG_WLR=1` and post log
4) is this a laptop

**Comment by Si-Ki** [comment]
1. No
2. was having `Seatd.sock` not found... error, asked a friend, added this, error gone.
3. log [here](https://0x0.st/oxLv.log) with `HYPRLAND_LOG_WLR=1` in [that](https://0x0.st/oxL_.sh) file
4. No but my CPU very old > `Intel i5 M 430 (4) @ 2.260GHz`
5. Sorry for being impatient and creating an issue of this discussion
